src/popup/popup.js

```diff
@@ -111,6 +118,33 @@ elements.clearFiltersButton.addEventListener('click', () => {
   elements.searchButton.click();
 });
 
+// a function to fetch licencse explanation
+const tooltiGen = () => {
+  const pathId = this.id;
+  const heading = tooltipInfo[pathId].tooltipHeading;
+  const content = tooltipInfo[pathId].tooltipContent;
+  let html = ``;
+  html += `<b>${heading}</b><p>${content}</p><br/><p class="caption is-pulled-right margin-small">Read more about the tool <a href='#'>here</a></p>`;
+  document.getElementById(`${pathId}-gen`).innerHTML = html;
+};
+// added event listeners in all tooltip btn
+elements.cc0btn.onclick = tooltiGen;
+elements.pdmbtn.onclick = tooltiGen;
+elements.bybtn.onclick = tooltiGen;
+elements.bysabtn.onclick = tooltiGen;
+elements.byncbtn.onclick = tooltiGen;
+elements.byndbtn.onclick = tooltiGen;
+elements.byncsabtn.onclick = tooltiGen;
+elements.byncndbtn.onclick = tooltiGen;
+elements.cc0btn.onmouseover = tooltiGen;
+elements.pdmbtn.onmouseover = tooltiGen;
+elements.bybtn.onmouseover = tooltiGen;
+elements.bysabtn.onmouseover = tooltiGen;
+elements.byncbtn.onmouseover = tooltiGen;
+elements.byndbtn.onmouseover = tooltiGen;
+elements.byncsabtn.onmouseover = tooltiGen;
+elements.byncndbtn.onmouseover = tooltiGen;
+
```

Comment:
Refactoring can be done as stated above. Also, we only want to toggle the tooltip on click and not hover because if the user hovers away onto the tooltip to click the link, the tooltip will disappear.